source/browseMode.py

```diff
@@ -1612,19 +1629,28 @@ def _getNearestTableCell(self, tableID, startPos, origRow, origCol, origRowSpan,
 		elif axis == "column":
 			destCol += origColSpan if movement == "next" else -1
 
-		if destCol < 1 or destRow<1:
-			# Optimisation: We're definitely at the edge of the column or row.
-			raise LookupError
-
-		return self._getTableCellAt(tableID,startPos,destRow,destCol)
+		# Try and fetch the cell at these coordinates, though  if a hidden cell is hit, try up to 4 more times moving the coordinates on by one cell each time
+		limit=5

```

Comment:
Please make this a constant, as it seems reasonable that something might want to tweak this.